### PR TITLE
feat(nuxt): storybook generators

### PR DESCRIPTION
**Pull request by mandarini**
* `@nx/nuxt:storybook-configuration`
* `@nx/nuxt:stories`

Invoking the `@nx/vue` generators, since Storybook does not officially support any nuxt-specific framework or module at the moment.

### CODE REVIEW

**Review by isaacplmann — APPROVED** [review]

**Review by Coly010 — APPROVED** [review]

File: packages/nuxt/src/generators/stories/schema.json [hunk 1]
```diff
@@ -0,0 +1,64 @@
+{
+  "$schema": "http://json-schema.org/schema",
+  "cli": "nx",
+  "$id": "NxVueStorybookStories",
```
Comment (JamesHenry) [inline packages/nuxt/src/generators/stories/schema.json:4]:
```suggestion
  "$id": "NxNuxtStorybookStories",
```

File: packages/nuxt/src/generators/stories/schema.json [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  "title": "Generate Vue Storybook stories",
```
Comment (JamesHenry) [inline packages/nuxt/src/generators/stories/schema.json:5]:
```suggestion
  "title": "Generate Nuxt Storybook stories",
```

File: packages/nuxt/src/generators/storybook-configuration/schema.json [hunk 3]
```diff
@@ -0,0 +1,77 @@
+{
+  "$schema": "http://json-schema.org/schema",
+  "cli": "nx",
+  "$id": "NxVueStorybookConfigure",
```
Comment (JamesHenry) [inline packages/nuxt/src/generators/storybook-configuration/schema.json:4]:
```suggestion
  "$id": "NxNuxtStorybookConfigure",
```

File: packages/nuxt/src/generators/storybook-configuration/schema.json [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+  "title": "Vue Storybook Configure",
```
Comment (JamesHenry) [inline packages/nuxt/src/generators/storybook-configuration/schema.json:5]:
```suggestion
  "title": "Nuxt Storybook Configure",
```

File: packages/nuxt/src/generators/storybook-configuration/schema.json [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+  "description": "Set up Storybook for a Vue project.",
```
Comment (JamesHenry) [inline packages/nuxt/src/generators/storybook-configuration/schema.json:6]:
```suggestion
  "description": "Set up Storybook for a Nuxt project.",
```

**Review by JamesHenry — CHANGES_REQUESTED** [review]
Mainly copy 🍝 issues and one point around globs, otherwise LGTM

**Comment by mandarini** [comment]
Fixed!

**Review by JamesHenry — APPROVED** [review]

**Review by ndcunningham — APPROVED** [review]
LGTM

**Pull request merged**